index.js

```diff
@@ -27,9 +27,18 @@ const getPool = () => {
 
 const createTables = async (pool) => {
     await pool.query(`
-        DROP TABLE IF EXISTS costumes;    
+        DROP TABLE IF EXISTS orders_costumes;
+        DROP TABLE IF EXISTS orders;  
+        DROP TABLE IF EXISTS customers;
+        DROP TABLE IF EXISTS costumes; 
     `)
     await pool.query(` 
+    CREATE TABLE customers(
+        id SERIAL PRIMARY KEY,
+        full_name VARCHAR(80) NOT NULL,
```

Comment:
I think this may be too few characters for fullname? esp. when you take into account i18n.

I think it's standard practice to just use 255 for most varchars (that way you can enforce stricter limits without them being locked in stone)